File: image.py
import sys
from PIL import Image
import numpy

numpy.set_printoptions(threshold=numpy.nan)

im = Image.open(sys.argv[1])
arr = numpy.array(im)
#print(arr)
print(arr[len(arr)-1][len(arr[len(arr)-1])-1][0])

File: image.cpp
#include <fstream>
#include <iostream>
using namespace std;

int main(int argc, char **argv) {
  ifstream image;

  //open input file------------------------------------------
  image.open(argv[1]);
  //check if the file was opened successfully
  if (!image.good()) {
    cerr<<"Opening of input file failed"<<endl;
    exit(1);
  }
  while (!image.eof()) {
    string nextLine;
    getline(image, nextLine);
    if (!nextLine.empty())
      cout << nextLine << endl;
  }
}
